data/json/itemgroups/SUS/domestic.json

```diff
@@ -618,8 +618,18 @@
       { "item": "soap_holder", "prob": 40 },
       { "item": "soap", "count": [ 1, 4 ], "prob": 90 },
       { "item": "razor_shaving", "count": [ 1, 2 ], "prob": 20 },
+      { "item": "cotton_ball", "prob": 50 },
       { "item": "bleach", "prob": 15 },
       { "item": "ammonia", "prob": 20 },
+      { "item": "eyedrops", "prob": 15 },
+      { "item": "saline", "prob": 10 },
+      { "item": "pepto", "prob": 50 },
+      { "item": "inhaler", "prob": 25 },
+      { "item": "dayquil", "prob": 30 },
+      { "item": "nyquil", "prob": 30 },
+      { "item": "vitamins", "prob": 30 },
+      { "item": "gummy_vitamins", "prob": 10 },
+      { "item": "calcium_tablet", "prob": 25 },
```

Comment:
This is a collection. Doing it like this makes it likely for large assortments of random medicines to spawn in sinks. Use a repeated distribution or something.